utils/ash_helpers.ps1

```diff
@@ -127,8 +127,7 @@ function Invoke-ASH {
                     '-t'
                     $AshImageName
                     "'$ashRoot'"
-                    $($buildArgs -join ' '
-                    '-q')
+                    $($buildArgs -join ' ')
```

Comment:
Is this line needed?

Reply:
@rafaelpereyra yes, so that additional args collected for the build are passed into the `docker build` call. The reason for the change is I saw that the `-q` flag was being passed in every time for this build call, resulting in no build output at all regardless if -Quiet was included.